components/Modal/Send.vue

```diff
@@ -272,12 +272,15 @@ export default {
       }
     },
     prefixValidation(address) {
-      if (address && address.startsWith(this.network.addressPrefix)) {
-        return true
-      } else {
-        this.addressError = `prefix does not match this network's prefix`
-        return false
+      if (address) {
+        for (let i = 0; i < this.network.allowedAddressPrefix.length; i += 1) {
+          if (address.startsWith(this.network.allowedAddressPrefix[i])) {
+            return true
+          }
+        }
```

Comment:
I think simpler would be `address  && this.network.allowedAddressPrefix.some(prefix => address.startsWith(prefix))`